slack-service/README.md

```diff
@@ -0,0 +1,112 @@
+# Self-Service Agent Slack Bot
+
+A Slack bot that provides self-service capabilities through AI agents, allowing users to interact with various services (like laptop refresh requests) directly within Slack.
+
+## Setup and Deployment
+
+Follow these steps to configure the Slack app and deploy the service.
+
+### Step 1: Create Your Slack App & Get Credentials
+
+You must create the Slack app first to get the **Bot Token**, which is required for the deployment.
+
+1.  **Create a New App:**
+
+    - Go to the [Slack API Console](https://api.slack.com/apps) and click **"Create New App"**.
+    - Choose **"From scratch"**, enter an app name (e.g., `Self-Service Agent`), and select your workspace.
+
+2.  **Configure Bot Token Scopes:**
+
+    - In the app's settings, go to **"OAuth & Permissions"**.
+    - Scroll down to **"Scopes"** -\> **"Bot Token Scopes"** and add the following permissions:
+      - `chat:write` (to send messages)
+      - `users:read` (to View messages and other content in direct messages that app has been added to)
+      - `im:history` (to see direct messages)
```

Comment:
I think we also want to get/use information about which user we are chatting with. Not sure what is available but I think there was email (unless there is something better) which would also require `users:read.email`